### ISSUE TITLE
Sometimes packed application on Windows can't start FFMpeg

### ISSUE THREAD

**Issue by CarlGao4** [CarlGao4/Demucs-Gui#37]
### happen

```
Traceback (most recent call last):
  File "audio.py", line 46, in checkFFMpeg
  File "shared.py", line 125, in Popen
  File "subprocess.py", line 992, in __init__
  File "subprocess.py", line 1365, in _get_handles
  File "subprocess.py", line 1416, in _make_inheritable
OSError: [WinError 50]
```

### What is the platform you are using?

Windows

**Comment by CarlGao4** [comment]
Will be fixed in 1.1

**Comment by CarlGao4** [comment]
Fixed in 1.0.1
